go-runtime/compile/testdata/one/one.go

```diff
@@ -9,6 +9,33 @@ import (
 	"github.com/TBD54566975/ftl/go-runtime/ftl"
 )
 
+//ftl:enum
+type Color string
+
+const (
+	Red   Color = "Red"
+	Blue  Color = "Blue"
+	Green Color = "Green"
+)
+
+//ftl:enum
+type ColorInt int
+
+const (
+	RedInt   ColorInt = 0
+	BlueInt  ColorInt = 1
+	GreenInt ColorInt = 2
+)
+
+//ftl:enum
+type SimpleInt int
+
+const (
+	Zero SimpleInt = iota
```

Comment:
Oh nice, this answers my previous question :)

Does it work with iota expressions like `iota * 2`?